chat-commands.js

```diff
@@ -3409,6 +3409,21 @@ exports.commands = {
 				group: targetUser.group,
 				rooms: roomList,
 			};
+			if (targetUser in room.users && user.can('addhtml', null, room)) {
```

Comment:
It can help with various features. Global bots are generally all maintained by globals who are known to be trustworthy with private information like that, I think.